mijia/Cargo.toml

```diff
@@ -13,4 +13,6 @@ dbus-tokio = "0.5.2"
 eyre = "0.6"
 futures = "0.3.5"
 itertools = "0.9.0"
+log = "0.4.11"
+pretty_env_logger = "0.4.0"
```

Comment:
Should these be dev dependencies if they're only used by examples?

Reply:
`log` is needed for the library, but `pretty_env_logger` you're right.